src/control/directive.ts

```diff
@@ -53,7 +53,7 @@ class ControlValueAccessorAdapter implements FormViewAdapter {
 
 @Directive({
   // tslint:disable-next-line:directive-selector
-  selector: '[ngrxFormControlState]',
+  selector: ':not([ngrxFormsAction])[ngrxFormControlState]',
```

Comment:
i'd prefer `[ngrxFormControlState]:not([ngrxFormsAction])`

Reply:
See https://github.com/MrWolfZ/ngrx-forms/pull/166#discussion_r349931342